### PR TITLE
chore: Removed devx commands 

### PR DESCRIPTION
**Pull request by Aerex**
# Context

The purpose of this PR is to remove all devx related commands since the backend services for these commands will be removed (see [announcement](https://www.ibm.com/cloud/blog/announcements/deprecation-of-ibm-cloud-starter-kits))

The commands that will be removed are as follows:

- [build](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli)
- [code](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#code)              
- [console](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#console)
- [create](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#create)
- [debug](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#debug)
- [delete](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#delete)
- [deploy](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#deploy)
- [diag](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#diag)
- [edit](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#edit)
- [get-credentials](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#get-credentials)
- [list](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#list)
- [run](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#run)
- [shell](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#shell)
- [status](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#status)
- [stop](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#stop)
- [test](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#test)
- [view](https://cloud.ibm.com/docs/cli?topic=cli-idt-cli#view)

### CODE REVIEW

File: src/cloudfoundry/LogsCommandManager.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2016, 2022
+ * Copyright IBM Corporation 2016, 2023
```
Comment (Aerex) [inline src/cloudfoundry/LogsCommandManager.ts:2]: **Callout:** Updated end date to the current year. Did the same thing for the other files.

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
     ```
     // A launch configuration that compiles the extension and then opens it inside a new window
     {
-        "version": "0.1.0",
+        "version": "0.2.0",
```
Comment (Aerex) [inline DEVELOPMENT.md:12]: **Callout:** Updated the new version for vscode.

File: README.md [hunk 3]
```diff
@@ -62,22 +64,19 @@ Easily invoke commands from the IBM Cloud CLI from directly inside of the VS Cod
...
-- Create a project using one of the two methods below:
-    - `ibmcloud dev create` using the IBM Cloud CLI (outside of VS Code)
-    - [IBM Cloud CLI Documentation](https://cloud.ibm.com/docs/apps?topic=apps-create-deploy-app-cli)
+- Create a project or open an existing project. Examples on how to create a project are listed below:
+    - [Create and deploy a Node.js Express app by using IBM Cloud Schematics](https://cloud.ibm.com/docs/apps?topic=apps-tutorial-node-webapp)
+    - [Create and deploy a Java Spring app by using IBM Cloud Schematics](https://cloud.ibm.com/docs/apps?topic=apps-tutorial-spring-webapp)
+    - [Create and deploy a Java Liberty app by using IBM Cloud Schematics](https://cloud.ibm.com/docs/apps?topic=apps-tutorial-liberty-webapp)
+    - [Create and deploy an app using Code Engine](https://cloud.ibm.com/docs/apps?topic=apps-tutorial-cd-code-engine)
+    - [Create and deploy a secure app with Devops](https://cloud.ibm.com/docs/apps?topic=apps-tutorial-cd-devsecops)
+    - [Develop a Kubernetes app with Helm](https://www.ibm.com/cloud/architecture/tutorials/use-develop-kubernetes-app-with-helm-toolchain)
```
Comment (Aerex) [inline README.md:73]: **Callout**: This is subject to change once we figure what we want here.

**Review by steveclay — DISMISSED** [review]

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -9,7 +9,7 @@
     ```
     // A launch configuration that compiles the extension and then opens it inside a new window
     {
-        "version": "0.1.0",
```
Comment (steveclay) [inline DEVELOPMENT.md:12]: Shouldn't this be `1.0.0` ?

File: package.json [hunk 5]
```diff
@@ -318,16 +243,10 @@
   "ibm": {
     "cli": {
       "command": "ibmcloud",
-      "version": "2.7.0",
+      "version": "2.18.0",
```
Comment (Aerex) [inline package.json:246]: **Callout**: Updated to the latest version of the CLI as of today

**Review by steveclay — APPROVED** [review]

**Pull request merged**